extensions/adobe/experience/mobile-lifecycle-details.schema.json

```diff
@@ -0,0 +1,288 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/mixins/mobile-lifecycle-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Mobile Lifecycle Details",
+  "type": "object",
+  "description": "Application lifecycle information recorded via an ExperienceEvent that is specific to the Mobile channel. Includes field definitions for automatically collected data by the AEP Edge mobile extension library.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "definitions": {
+    "mobile-lifecycle-details": {
+      "properties": {
+        "xdm:mobileApplication": {
+          "title": "Mobile Application",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/mobile-application",
+          "description": "Application information related to an ExperienceEvent, including the name of the application, app version, launches, and closures."
+        },
+        "xdm:device": {
+          "title": "Device",
+          "$ref": "https://ns.adobe.com/xdm/context/device",
+          "description": "Information about the mobile device the application lifecycle event observation occurred in, including device type and model."
+        }
+      }
+    },
+    "environment": {
```

Comment:
@kevinlind , as you mentioned that you do not want to include the browserdetails properties from environment, this definition here will bring that as its referring the "environment" data type. Is that a requirement ?

Reply:
@mjasrotia Thanks for pointing this out. No, that is not a requirement. Trying to define specific fields from a datatype into a mixin is new to me, so I'm a little unclear on the format. The end result should be what I outlined in `mobile-lifecycle-details.example.1.json` where I can use specific environment fields, but not browserDetails.
```
{
  "xdm:environment": {
    "xdm:type": "application",
    "xdm:carrier": "Verizon",
    "xdm:operatingSystem": "iOS",
    "xdm:operatingSystemVersion": "14.2",
    "xdm:operatingSystemVendor": "Apple",
    "dc:language": "en-US"
  }
}
```

I'll remove this from the PR.